packages/common/src/services/remote-config/types.ts

```diff
@@ -129,6 +129,26 @@ export enum IntKeys {
    */
   MAX_AUDIO_PURCHASE_AMOUNT = 'MAX_AUDIO_PURCHASE_AMOUNT',
 
+  /**
+   * Minimum price for purchasable content in cents
+   */
+  MIN_CONTENT_PRICE_CENTS = 'MIN_CONTENT_PRICE_CENTS',
```

Comment:
I know it seems a little over the top to define the limits for content and purchase separately. But I figured we may want to allow them to change independently since the usdc limit is based on onramp provider and is a function or content price, pay extra amount, and existing balance.
At the moment, both of these pairs of limits have the same values in remote config.
So if you upload a track at maximum price, users won't be able to pay extra unless they have a balance already.

Reply:
yeah it makes sense